scenariolib/visit.go

```diff
@@ -67,23 +67,19 @@ func NewVisit(_searchtoken string, _uatoken string, _useragent string, language
 
 	v.WaitBetweenActions = !c.DontWaitBetweenVisits
 	v.Anonymous = false
-	if c.AllowAnonymous {
```

Comment:
I don't believe the default value for anonymous threshold was ever publicized. All the examples I have and have shared always had the 2 params. So in this case I think we can be fine :)

Thanks for pointing it out though. Nice observation.

(And if someone still has it in its config, then it will just be ignored)